components/CollaboratorInput/SubmitForm.tsx

```diff
@@ -60,18 +67,28 @@ const SubmitForm = () => {
         Collaborator {currentCollaborator + 1}:
       </p>
       <CollaboratorValues />
-      <Button
-        className="mt-10 mb-0 mx-auto md:mx-[unset] relative z-[2] md:w-52 md:h-12 w-36 h-12 font-normal md:bg-transparent bg-[#AC444475] border-[#E18583] text-[12px]"
-        onClick={handleSubmit}
-        disabled={
-          Boolean(!legalName) ||
-          Boolean(!email) ||
-          Boolean(!split) ||
-          isContributor
-        }
-      >
-        SUBMIT
-      </Button>
+      <div className="mt-10 flex items-center gap-4">
+        {currentCollaborator > 0 && (
+          <Button
+            className="mb-0 mx-auto md:mx-[unset] relative z-[2] md:w-52 md:h-12 w-36 h-12 font-normal md:bg-transparent bg-[#AC444475] border-[#E18583] text-[12px]"
+            onClick={() => setCurrentCollaborator(currentCollaborator - 1)}
+          >
+            BACK
+          </Button>
+        )}
+        <Button
+          className="mb-0 mx-auto md:mx-[unset] relative z-[2] md:w-52 md:h-12 w-36 h-12 font-normal md:bg-transparent bg-[#AC444475] border-[#E18583] text-[12px]"
+          onClick={handleSubmit}
+          disabled={
+            Boolean(!legalName) ||
+            Boolean(!email) ||
+            Boolean(!split) ||
+            isContributor
+          }
+        >
+          SUBMIT
+        </Button>
+      </div>
```

Comment:
Could you please make this as a new component?
- actual: adding new HTML to an existing component.
- expected: Create a new, standalone, component for `<ActionButtons />` or just for the new `<BackButton />` in a new component file `ActionButtons.tsx`